onward/app/controllers/MostPopularController.scala

```diff
@@ -37,6 +37,7 @@ object MostPopularController extends Controller with Logging with ExecutionConte
         case popular => Cached(900) {
           JsonComponent(
             "html" -> views.html.fragments.collections.popular(popular),
+            "ABhtml" -> views.html.fragments.collections.popularABTest(popular),
```

Comment:
alternatively you could add a new route with a slightly different url and get the AB test to fetch that one instead